image-filters/image-filters.markdown

```diff
@@ -652,9 +649,9 @@ for(int px in pixels) {
 
 <markdown>
 At the end we can print this hue to the screen, or display it next to the
-picture (\aosafigref{500l.imagefilters.hueranges}).
+picture (\aosafigref{500l.imagefilters.hueranges}). 
 
-\aosafigure[240pt]{image-filters-images/hueranges.jpg}{Dominant hue vs. # buckets used in hue range}{500l.imagefilters.hueranges}
+\aosafigure[240pt]{image-filters-images/hueranges.jpg}{Dominant hue versus number of buckets used in hue range}{500l.imagefilters.hueranges}
```

Comment:
I don't really understand this image. What do the numbers refer to? Would it be possible to write an expanded explanation of the image?